packages/riipen-ui/src/components/TableBody.test.jsx

```diff
@@ -0,0 +1,78 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import TableBody from "./TableBody";
+import Table from "./Table";
+
+describe("<TableBody>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(
+        <Table>
+          <TableBody />
+        </Table>
+      );
+    } catch (theError) {
+      error = theError;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(
+      <Table>
+        <TableBody />
+      </Table>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = <td>Test</td>;
+      const childContent = <tr>{child}</tr>;
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>{childContent}</TableBody>
+        </Table>
+      );
+
+      expect(wrapper.containsMatchingElement(childContent)).toBeTruthy();
+    });
+  });
+
+  describe("classes prop", () => {
+    it("applies all classes to the root node", () => {
+      const classes = ["classOne"];
+
+      const wrapper = mount(
+        <Table>
+          <TableBody classes={classes} />
+        </Table>
+      );
+
+      expect(wrapper.find("tbody").props().classes).toContain(classes[0]);
+    });
```

Comment:
it applies all classes to the tbody node instead of root node?

Reply:
Adjusted the test name